src/iop/basecurve.c

```diff
@@ -424,6 +449,13 @@ int gauss_reduce_cl(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece,
 }
 
 
+static float exposure_increment(float stops, int e, float fusion, float bias)
+{
+  float offset = (stops * fusion) * (bias - 1.0) / 2.0;
+  return powf(2.0f, stops * e + offset);
```

Comment:
do i understand correctly that this leaves the exposure fusion offsets untouched, but adds an exposure bias, equivalent to applying the exposure module before this manually? the difference being that it's scaled by the exposure stops, so the interface is a bit more intuitive (and convenient, because it's right there in the same module)?